dataprofiler/profilers/profile_builder.py

```diff
@@ -792,6 +813,14 @@ def tqdm(l):
         for col in tqdm(df.columns):
             self._profile[col].update_column_profilers(
                 clean_sampled_dict[col], pool)
+
         if pool is not None:
-            pool.close() # Close pool for new tasks
-            pool.join() # Wait for all workers to complete
+            pool.close()  # Close pool for new tasks
+            pool.join()  # Wait for all workers to complete
+
+        # Only pass along sample ids if min col samples exists
+        samples_for_row_stats = None
+        if self._min_col_samples_used > 0:
```

Comment:
`if (min_true_samples is None or min_true_samples == 0) and len(sample_ids) > 0:`